fiatdao/fiat-ui
# Close position instructions

## Issue ticket number & link
Unclear how to close position

## QA Instructions
ensure you see this on test build

![image](https://user-images.githubusercontent.com/97852365/182852107-91b4ecbd-312d-4392-866f-4ca48746905f.png)
